android/tangram/src/com/mapzen/tangram/MapController.java

```diff
@@ -552,6 +555,12 @@ public void cancelUrlRequest(String url) {
         okClient.cancel(url);
     }
 
+    public String getFontFilePath(String family, String weight, String style) {
```

Comment:
Let's not put this in the middle of a group of methods explicitly labelled as 'networking'